cmd/terramate/cli/project.go

```diff
@@ -192,14 +214,20 @@ func (p *project) setDefaults() error {
 
 	if gitOpt.DefaultBranchBaseRef == "" {
 		gitOpt.DefaultBranchBaseRef = defaultBranchBaseRef
+	} else {
+		p.git.baseRefConfigured = true
 	}
```

Comment:
what do you think?
```suggestion
    p.git.baseRefConfigured = gitOpt.DefaultBranchBaseRef != ""
	if !p.git.baseRefConfigured {
		gitOpt.DefaultBranchBaseRef = defaultBranchBaseRef
	}
```
if you like then please apply also to the others.